wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/CreateSecureGuestLink/CreateSecureConversationGuestLinkViewModel.swift

```diff
@@ -91,6 +91,7 @@ final class CreateSecureConversationGuestLinkViewModel {
             switch result {
             case .success(let link?):
                 self.delegate?.viewModel(self, didCreateLink: link)
+                NotificationCenter.default.post(name: .didCreateSecureGuestLink, object: nil, userInfo: ["link": link])
```

Comment:
Let's also use a constant key for `"link"` to have the value equal for sending and receiving.

Reply:
Done! https://github.com/wireapp/wire-ios/commit/3795b5e773dd53b39e480d870ad35d4eaf04591f